reference/tax_settings.v3.yml

```diff
@@ -146,7 +146,19 @@ components:
         should_subtract_store_tax:
           default: true
           type: boolean
-          description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.    
+          description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.
+        should_use_geolocation_to_determine_guest_shopper_tax_zone:
+          default: false
+          type: boolean
+          description: Configures the tax estimation and display settings the shoppers see based on their geolocation before they confirm the destination address at checkout.
```

Comment:
Ah, I thought we were missing something - thanks for including it.

This description could be improved. I note we speak to "When enabled," in the above `should_subtract_store_tax` description, which I think would be a good starting point.

Reply:
added in `When enabled`